_layouts/home.html

```diff
@@ -31,33 +33,28 @@ <h3>
               {{ post.title | escape }}
             </a>
           </h3>
-          {%- if site.show_excerpts -%}
-            {{ post.excerpt }}
-          {%- endif -%}
+          <div class="post-snippet">
+            {{ post.content | strip_html | truncatewords: 22}}
```

Comment:
For this PR, lets set the truncation to *40 words*.

Ideally this should be the opening paragraph - since the intention is it to be the hook for the entire article.  But that is outside the scope of this PR.

Reply:
This was made as a stylistic decision. On the home page we can dictate how many/which words are shown.

However, when using Google Custom Search you are provided web results in the following format:
![Screen Shot 2021-04-15 at 2 05 52 pm](https://user-images.githubusercontent.com/7382785/114821821-b8ebb180-9df3-11eb-8919-cbde4eae7837.png)

Currently Google dictates which and how many words are shown in the snippet. I am unaware of a way for us to show more words. I had a look at the search results and they roughly returned 22 words. Therefore, for look and feel consistency I decided that that would be the correct amount to show on the home page.

Reply:
:+1: 

Cool - good justification.  And a great example of the real value in code reviews.

Leave as is - we can re-visit later if needs be.

On a side note, my quick search was coming back with results around 160 chars (which is ~22 words ish) for description meta.  So it might be related to that.